paymaster-analysis/alchemy/alchemy-erc721.ts

```diff
@@ -0,0 +1,289 @@
+import dotenv from "dotenv";
```

Comment:
If feels like there is a lot of duplicated code between the `alchemy-*.ts` scripts (from a quick glance, only the `callData` value and some of the `initCode` setup are any different)

Reply:
Done 👍🏾